Websites/browserbench.org/MotionMark1.2/tests/bouncing-particles/resources/bouncing-canvas-images.js

```diff
@@ -49,7 +49,7 @@ BouncingCanvasImagesStage = Utilities.createSubclass(BouncingCanvasParticlesStag
     initialize: function(benchmark, options)
     {
         BouncingCanvasParticlesStage.prototype.initialize.call(this, benchmark, options);
-        var imageSrc = options["imageSrc"] || "resources/yin-yang.svg";
+        var imageSrc = options["imageSrc"] || "../resources/yin-yang.svg";
```

Comment:
Does this actually work? Because there's still a `../`. Or is that OK because it's not in query any more?

Reply:
It does work. I've worked with Ling to run a test version of the same server, and verified that it does, in fact, fix the problem. The problem doesn't seem to be relative paths themselves, but just when `../` paths appear _inside query parameters_.